app/views/ExposureHistory/NextSteps.tsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
-import { TouchableOpacity, View, StyleSheet } from 'react-native';
+import { TouchableOpacity, View, StyleSheet, Linking } from 'react-native';
 import { useNavigation } from '@react-navigation/native';
 import { useTranslation } from 'react-i18next';
+import { SvgXml } from 'react-native-svg';
 
 import { Typography } from '../../components/Typography';
 import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
 import { Screens, useStatusBarEffect } from '../../navigation';
 
 import { Buttons, Spacing, Typography as TypographyStyles } from '../../styles';
+import { Icons } from '../../assets';
 
-import { AUTHORITY_NAME as healthAuthorityName } from '../../constants/authorities';
+import {
+  AUTHORITY_NAME as healthAuthorityName,
+  AUTHORITY_ADVICE_URL,
+} from '../../constants/authorities';
+import { Colors } from 'react-native/Libraries/NewAppScreen';
```

Comment:
Why are we using these colors? should we be using the common styles.

Reply:
Whoops, VS code auto-import... 🤦